R/pkg/R/functions.R

```diff
@@ -4458,3 +4469,25 @@ setMethod("timestamp_seconds",
             )
             column(jc)
           })
+
+#' @details
+#' \code{vector_to_array} Converts a column of MLlib sparse/dense vectors into
+#' a column of dense arrays.
+#'
+#' @param dtype The data type of the output array. Valid values: "float64" or "float32".
+#'
+#' @rdname column_ml_functions
+#' @aliases vector_to_array vector_to_array,Column-method
+#' @note vector_to_array since 3.1.0
```

Comment:
Can we add an example ( `dontrun`) while we're making a followup to address https://github.com/apache/spark/pull/29917/files#r499340542?